schemas/deviceToCloud/ground_fix/ground-fix-example.json

```diff
@@ -0,0 +1,36 @@
+{
+    "appId": "GROUND_FIX",
+    "messageType": "DATA",
+    "data": {
+        "doReply": false,
+        "wifi": {
```

Comment:
Are both "wifi" and "lte" objects always required?

Reply:
No, just one or the other. The `minimumPropertiesRequired` prop in the schema hints at that, but it's not very obvious.